resources/settings/settings.xml

```diff
@@ -88,6 +88,7 @@
 			<listEntry value="10">@Strings.SunriseSunset</listEntry>
 			<listEntry value="11">@Strings.Weather</listEntry>
 			<listEntry value="13">@Strings.Humidity</listEntry>
+			<listEntry value="14">@Strings.PulseOx</listEntry>
```

Comment:
Crystal should only show settings that are actually supported by a given device (see how e.g. Temperature and Floors are handled). `currentOxygenSaturation` is [only supported](https://developer.garmin.com/connect-iq/api-docs/Toybox/Activity/Info.html#currentOxygenSaturation-var) by quite a limited set of devices - is there an easy way we can hide the setting from other devices?

Reply:
I'll check

Reply:
Do you mean like for example the resource file resources-approachs62 has "listEntry value="1">@Strings.FloorsClimbed</listEntry" in comments? If so, would that mean a specific resource file for each device that supports Pulse Ox?

Reply:
Because it's supported by quite a few devices now:

Captain Marvel
D2™ Air
Darth Vader™
Descent™ Mk2 / Descent™ Mk2i
Descent™ Mk2 S
Enduro™
First Avenger
Forerunner® 245
Forerunner® 245 Music
Forerunner® 745
Forerunner® 945
Forerunner® 945 LTE
fēnix® 5X Plus
fēnix® 6 / 6 Solar / 6 Dual Power
fēnix® 6 Pro / 6 Sapphire / 6 Pro Solar / 6 Pro Dual Power / quatix® 6
fēnix® 6S / 6S Solar / 6S Dual Power
fēnix® 6S Pro / 6S Sapphire / 6S Pro Solar / 6S Pro Dual Power
fēnix® 6X Pro / 6X Sapphire / 6X Pro Solar / tactix® Delta Sapphire / Delta Solar / Delta Solar - Ballistics Edition / quatix® 6X / 6X Solar / 6X Dual Power
MARQ™ Adventurer
MARQ™ Athlete
MARQ™ Aviator
MARQ® Captain / MARQ® Captain: American Magic Edition
MARQ™ Commander
MARQ™ Driver
MARQ™ Expedition
MARQ™ Golfer
Rey™
Venu™
Venu™ 2
Venu™ 2S